statistics/tiny/stats_with_epoch_change-1.json

```diff
@@ -0,0 +1,431 @@
+                            "beefy_root": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                            "lookup_anchor": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                            "lookup_anchor_slot": 0,
+                            "prerequisites": []
+                        },
+                        "core_index": 0,
+                        "authorizer_hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                        "auth_output": "0x",
+                        "segment_root_lookup": [],
+                        "results": [
+                            {
+                                "service_id": 0,
+                                "code_hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                                "payload_hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                                "gas": 0,
+                                "result": {
+                                    "ok": "0x"
+                                }
+                            }
+                        ]
+                    },
+                    "slot": 0,
+                    "signatures": [
+                        {
+                            "validator_index": 2,
+                            "signature": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+                        },
+                        {
+                            "validator_index": 3,
+                            "signature": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+                        }
+                    ]
+                }
+            ],
+            "assurances": [
+                {
+                    "anchor": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "bitfield": "0x00",
+                    "validator_index": 0,
+                    "signature": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+                },
+                {
+                    "anchor": "0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "bitfield": "0x00",
+                    "validator_index": 1,
+                    "signature": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+                }
+            ],
+            "disputes": {
+                "verdicts": [],
+                "culprits": [],
+                "faults": []
+            }
+        },
+        "reporters": []
```

Comment:
This is wrong, since there are a number of `guarantee` extrinsic present in the input. 

It would be best to not provide this value, and let it be deduced from input.

In other files where the values are correct, I have ignored `reporters` and calculated it from the extrinsic.

Reply:
should be fixed https://github.com/w3f/jamtestvectors/pull/28/commits/12505a1b7debdeeb582a09c0ab3673b8b9bb0e39

Reply:
Since the reporters set was computed from guarantee STF, shouldn't assume that we have it for statistics STF?